packages/vuetify/src/components/VSkeletonLoader/VSkeletonLoader.sass

```diff
@@ -204,10 +204,14 @@
 
   &__image
     height: $skeleton-loader-image-height
+    border-radius: 0
 
-    &:not(:first-child):not(:last-child)
+    ~ .v-skeleton-loader__card-heading
       border-radius: 0
 
+    &:first-child, &:last-child
```

Comment:
I've updated our Coding Guidelines to reflect our current procedure for pseudo elements. Could you please update to reflect? Thank you: https://www.notion.so/vuetify/Styles-d338e11a75d846658c4431c567efb54a